packages/neuron-ui/src/components/BroadcastTransaction/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { isSuccessResponse, RoutePath, useDidMount, useGoBack } from 'utils'
+import Dialog from 'widgets/Dialog'
+import AlertDialog from 'widgets/AlertDialog'
+import { useDispatch, useState as useGlobalState } from 'states'
+import { broadcastTransaction, getCurrentWallet, OfflineSignStatus } from 'services/remote'
+import { ReactComponent as HardWalletIcon } from 'widgets/Icons/HardWallet.svg'
+
+import styles from './broadcastTransaction.module.scss'
+
+const BroadcastTransaction = () => {
+  const {
+    app: { loadedTransaction = {} },
+  } = useGlobalState()
+
+  const [wallet, setWallet] = useState<State.Wallet | null>(null)
+  const [isBroadCasting, setIsBroadcasting] = useState(false)
+  const [t] = useTranslation()
+  const dispatch = useDispatch()
+  const [errMsg, setErrMsg] = useState('')
+
+  const { filePath, json } = loadedTransaction
+
+  const jsonContent = useMemo(() => {
+    return JSON.stringify(json, null, 2)
+  }, [json])
+
+  const signStatus: OfflineSignStatus = json.status
+
+  const isSigned = useMemo(() => signStatus === OfflineSignStatus.Signed, [signStatus])
+
+  const onBack = useGoBack()
+
+  const navigate = useNavigate()
+  const onBroadcast = useCallback(async () => {
+    setIsBroadcasting(true)
+    try {
+      const res = await broadcastTransaction({
+        ...json,
+        walletID: wallet!.id,
```

Comment:
Unlike `broadcast in offline sign dialog`, the transaction to broadcast may not belong to the current wallet because it's been signed.